examples/src/main/scala/za/co/absa/enceladus/examples/CustomRuleSample4.scala

```diff
@@ -27,6 +26,8 @@ import za.co.absa.enceladus.model.Dataset
 import za.co.absa.enceladus.utils.time.TimeZoneNormalizer
 
 object CustomRuleSample4 {
+  TimeZoneNormalizer.normalizeJVMTimeZone()
```

Comment:
Why is this one different from the other Samples? I'd suggest making them more like this rather than using `normalizeAll`

Reply:
I see the _Samples_ as increasing in complexity. So early on just one call, later how it's done in rest of the project.
Also shows two possible ways how to call set the whole system to normalized time zone.

Reply:
Overloading the `normalizeAll` method would be probably nice so you don't have to `Seq(X)` or call a different method.

```scala
def normalizeAll(sparks: Seq[SparkSession]): Unit = ???
def normalizeAll(sparks: SparkSession): Unit = ???
def normalizeAll(): Unit = ???
```

Reply:
Yeah, I was considering the second one. Third one is a synonym to `normalizeJVMTimeZone`

Reply:
Yeah, having aliases. I still like that.

Reply:
> I see the _Samples_ as increasing in complexity. So early on just one call, later how it's done in rest of the project.
> Also shows two possible ways how to call set the whole system to normalized time zone.

Why even have `normalizeAll` if that is never really used? If using it can lead to unexpected problems that we avoid by eagerly doing `normalizeJVMTimeZone`, why give the option and provide misleading examples that we would not use ourselves?

Reply:
> 
> 
> Why even have `normalizeAll` if that is never really used? If using it can lead to unexpected problems that we avoid by eagerly doing `normalizeJVMTimeZone`, why give the option and provide misleading examples that we would not use ourselves?

It's actually used in _Menas_ as well.